lms/static/scripts/frontend_apps/components/dashboard/DashboardActivityFilters.tsx

```diff
@@ -25,7 +30,12 @@ export default function DashboardActivityFilters({
   onAssignmentsChange,
   selectedStudentIds,
   onStudentsChange,
+  onSelectionReset,
```

Comment:
The UI uses the term "clear" but internal identifiers use the term "reset". Perhaps this should be `onClearSelection`?